packages/nextjs/components/scaffold-stark/Address.tsx

```diff
@@ -96,8 +96,8 @@ export const Address = ({
     );
   }
 
-  if (!validateChecksumAddress(checkSumAddress)) {
-    return <span className="text-error">Wrong address</span>;
+  if (!checkSumAddress || !/^0x[0-9a-fA-F]+$/.test(checkSumAddress)) {
```

Comment:
Possible to wrap these regex into an util function?

Reply:
I'll wrap the regex into a utility function as suggested. Thanks for pointing it out!